core/embed/rust/src/ui/display.rs

```diff
@@ -227,6 +748,42 @@ impl Font {
     pub fn line_height(self) -> i32 {
         constant::LINE_SPACE + self.text_height()
     }
+
+    pub fn get_glyph(self, ch: char) -> Option<Glyph> {
+        let gl_data = display::get_char_glyph(ch, self.0);
+
+        if gl_data.is_null() {
+            return None;
+        }
+
+        unsafe {
```

Comment:
this whole block should probably be moved to something like `pub unsafe fn Glyph::load(gl_data: *const u8) -> Self`
then maybe we don't even need the `Glyph::new` function, because `load`ing a Glyph is the only supported way to get one

Reply:
done in 3fa8513c9ca6f91afac8465beaec310815863f2c